Makefile

```diff
@@ -98,3 +98,8 @@ packages:
 .PHONY: upload-packages
 upload-packages:
 	./scripts/upload-packages.sh
+
+# Update vendoring
+.PHONY: vendor-update
+vendor-update:
+	glide update --strip-vendor
```

Comment:
shouldn't we add `glide update -v` before too?

Reply:
According to odo doc, this should be sufficient I guess.
I was thinking of adding `glide-vc --only-code --no-tests` as well 
WDYT ?

Reply:
I don't think we're adding that anymore. Using `glide-vc` and all.

Maybe we should echo after vendor-update saying that the user should `git add .` and do a separate commit? Similar to how `developers.md` describes it.

Reply:
using `glide-vc` removes extra clutter which solves https://github.com/redhat-developer/odo/issues/670 as well

Reply:
I believe there's a reason why we don't use glide-vc, @kadel would know once he gets back.

Reply:
Got it, so as of now, we can merge this in

Reply:
agreed. this LGTM.